backend/app/models/buildings.py

```diff
@@ -0,0 +1,30 @@
+from . import db
+from sqlalchemy import inspect, cast, String
+from sqlalchemy.orm.properties import ColumnProperty
+
+
+class Buildings(db.Model):
+    __tablename__ = "buildings"
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    address = db.Column(db.String, nullable=False)
+    name = db.Column(db.String, nullable=False)
+
```

Comment:
so SQLalchemy has this thing called `relationships` that make accessing tables through the ORM really easy - we'd wanna make one here for `log_records` and `residents`:

```
log_record = db.relationship('LogRecords', back_populates='building')
resident = db.relationship('Residents', back_populates='building')
```